### PR TITLE
reorganize ddboost replication information.

### PR DESCRIPTION
**Pull request by mkiyama**
--move replication info. into separate topic.
--update toc

Link to new topic in temporary review doc site.
http://docs-msk-gpdb6-dev.cfapps.io/7-0/admin_guide/managing/replication-ddb.html

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/managing/replication-ddb.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            href="../../utility_guide/ref/gpbackup_manager.xml">gpbackup_manager</xref></codeph>
+        with the DD Boost storage plugin to replicate a backup from one Data Domain system to a
+        second, remote, Data Domain system for disaster recovery. You can replicate a backup as part
+        of the backup process, or replicate an existing backup set as a separate operation. Both
+        methods require a <xref href="backup-ddboost-plugin.xml#topic_ddb-plugin/ddb-plugin-config"
+          >DD Boost configuration file</xref> that includes options that specify Data Domain system
+        locations and DD Boost configuration. The DD Boost storage plugin replicates the backup set
+        on the remote Data Domain system with DD Boost managed file replication. </p>
+      <ul id="ul_n4s_rrq_nkb">
+        <li>The <codeph>gpbackup</codeph> utility can replicate a backup set as part of the backup
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/replication-ddb.xml]: Just a suggestion, but it seems like these list items would be better off as separate sections (with titles).  It also seems like they could be formatted as stepped procedures with examples to make the process clearer. Maybe as a follow-on story?

File: gpdb-doc/dita/admin_guide/managing/replication-ddb.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <li>The <codeph>gpbackup_manager replicate-backup</codeph> command can replicate an existing
+          backup set created by <codeph>gpbackup</codeph>.<p>To replicate an existing backup set,
+            run <codeph>backup_manager replicate-backup</codeph> and specify a DD Boost
+            configuration file. The configuration file contains the same type of information that is
+            in the configuration file used to replicate a backup set with <codeph>gpbackup</codeph>.
+            </p><p>When using the <codeph>gpbackup_manager replicate-backup</codeph> command, the
+            configuration file <codeph>replication</codeph> option is ignored. The command always
+            attempts to replicate a back up.</p></li>
+      </ul>
+      <p>When replicating a backup, the Data Domain system where the backup is stored must have
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/replication-ddb.xml]: This info seems like it should be moved up to just after the intro paragraphs, maybe as a "prerequisites" section.  The restore sentence could either be moved to end of the overview paragraph or made into a section with steps as well.

File: gpdb-doc/dita/admin_guide/managing/replication-ddb.xml (hunk 1 above)
Comment (mkiyama) [reply gpdb-doc/dita/admin_guide/managing/replication-ddb.xml]: created sections for information

File: gpdb-doc/dita/admin_guide/managing/replication-ddb.xml (hunk 2 above)
Comment (mkiyama) [reply gpdb-doc/dita/admin_guide/managing/replication-ddb.xml]: moved information.

**Pull request merged**